docs/content/docs/build/container-states-events/index.md

```diff
@@ -96,7 +96,7 @@ But the new container has a different ID from the deleted one and subsequent cli
 
 #### Handling publication status
 
-Your code can test for the publication status with the [container.AttachState]({{< apiref "fluid-static" "IFluidContainer" "interface" "v2" >}}#attachstate-property) property which has an [AttachState]({{< packageref "container-definitions" "v2" >}}#attachstate-enum) value.
+Your code can test for the publication status with the [container.AttachState]({{< apiref "fluid-static" "IFluidContainer" "interface" "v2" >}}#attachstate-propertysignature) property which has an [AttachState]({{< packageref "container-definitions" "v2" >}}attachstate-enum) value.
```

Comment:
Same recommendation as https://github.com/microsoft/FluidFramework/pull/21768/files#r1664554640

Reply:
Updated and validated locally.